packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -17,4 +17,20 @@ describe("<DateRangeInput>", () => {
         const component = mount(<DateRangeInput />);
         expect(component.find(InputGroup).length).to.equal(2);
     });
+
+    it("startInputProps.inputRef receives reference to HTML input element", () => {
+        const inputRef = sinon.spy();
+        // full DOM rendering here so the ref handler is invoked
+        mount(<DateRangeInput startInputProps={{ inputRef }} />);
+        expect(inputRef.calledOnce).to.be.true;
+        expect(inputRef.firstCall.args[0]).to.be.an.instanceOf(HTMLInputElement);
```

Comment:
This pattern is taken from `buttonTests.tsx`. It's just ensuring that the component is actually passing an input element through to the ref handler we're providing via props (we weren't doing this in an earlier version of PR #614; these tests guarantee that we do).